app/controllers/HelpGuideController.scala

```diff
@@ -0,0 +1,14 @@
+package controllers
+
+import auth.OidcSecurity
+import javax.inject.{Inject, Singleton}
+import org.pac4j.play.scala.SecurityComponents
+import play.api.i18n.I18nSupport
+import play.api.mvc.{Action, AnyContent, Request}
+
+@Singleton
+class HelpGuideController @Inject()(val controllerComponents: SecurityComponents) extends OidcSecurity with I18nSupport  {
+  def helpGuide(): Action[AnyContent] = secureAction { implicit request: Request[AnyContent] =>
```

Comment:
Given how similar the pages are, and the fact that the controllers don't do anything complicated, I'd be inclined to merge the `helpGuide` and `accessibilityStatement` into a single controller. Maybe call it `HelpController`?

Reply:
Sounds good, I'll have a look into changing that, thank you!